migration/src/main/resources/META-INF/rewrite/axon-jakarta-47.yml

```diff
@@ -3,6 +3,10 @@ name: org.axonframework.migration.UpgradeAxonFramework_4_7_Jakarta
 displayName: Upgrade to Axonframework 4.7 Jakarta
 description: Migration file to upgrade from an Axon Framework Javax-specific project to Jakarta.
 recipeList:
+  - org.openrewrite.maven.UpgradeDependencyVersion:
+      groupId: org.axonframework
+      artifactId: "*"
+      newVersion: 4.x
```

Comment:
OpenRewrite supports a few [different version selectors](https://docs.openrewrite.org/reference/dependency-version-selectors#xrange); this one would resolve to 4.7.2 currently, but as soon as 4.8 becomes available, would resolve to that.. It would not select 5.0 once that comes out, as it stays within the 4.x range.

Didn't think it would make sense to pin this to 4.7.x, as there's no plans to do another point release there right? There's likely little harm in picking up 4.8+ with this same command, although it perhaps looks slightly odd in files suffixed `-47.yml`.

I don't suppose 4.8 would contain breaking changes, but if that does we would typically daisy chain the 4.7 migration before running any 4.8 migration steps, and then this version bump would not look out of place.